asset/script/main.js

```diff
@@ -188,5 +184,46 @@ form.addEventListener('submit', (event) => {
     showSuccessMsg();
     form.submit();
     clearField();
+    localStorage.removeItem('storedObj');
   }
 });
+
+// store form data object in localstorage
+const storeData = (obj) => {
+  const jsonObj = JSON.stringify(obj);
+  localStorage.setItem('storedObj', jsonObj);
+};
+
+// store form data in the localstorage
+const inputFieldHandler = (event) => {
+  const inputUserName = userNameField.value;
+  const inputUserEmail = emailField.value;
+  const inputUserMessage = messageField.value;
+  // form object
+  const newObj = {
+    name: inputUserName,
+    email: inputUserEmail,
+    message: inputUserMessage,
+  };
+  storeData(newObj);
+};
+userNameField.addEventListener('keypress', inputFieldHandler);
+emailField.addEventListener('keypress', inputFieldHandler);
+messageField.addEventListener('keypress', inputFieldHandler);
```

Comment:
- [ ] When a user enters a value and refreshes it shows up but when a user clears all the values and refreshes, those values still show up. Please ensure that the input fields are empty when a user clears all inputted values and refreshes the page.

Hint
You can use the `change` eventlistener. You can read more about it [here](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event)